data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsSourceConfig.java

```diff
@@ -56,6 +57,12 @@ public class RdsSourceConfig {
     @JsonProperty("acknowledgments")
     private boolean acknowledgments = false;
 
+    @JsonProperty("s3_data_file_acknowledgment_timeout")
+    private Duration dataFileAcknowledgmentTimeout = Duration.ofMinutes(5);
```

Comment:
What is the avg time taken to process data file ? Is this timeout good enough ?

Reply:
It's pretty fast in my test but it was not a large dataset. I guess we will need to find out and update accordingly. Will put a larger number here.

I see documentDb uses 2 hours by default. Wondering how was that number calculated?